hail/python/hail/backend/py4j_backend.py

```diff
@@ -75,15 +75,22 @@ def _parse_value_ir(self, code, ref_map={}, ir_map={}):
     def _to_java_value_ir(self, ir):
         pass
 
-    def register_ir_function(self, name, type_parameters, argument_names, argument_types, return_type, body):
+    def register_ir_function(self,
+                             name,
```

Comment:
Is it worth adding types here as well?

Reply:
I'm pretty sure mypy finds and uses the types from the super method, so it's not necessary for type checking, but I will add for documentary purposes.